app/workers/attachments/finish_direct_upload_job.rb

```diff
@@ -32,8 +32,8 @@ class Attachments::FinishDirectUploadJob < ApplicationJob
   queue_with_priority :high
 
   def perform(attachment_id)
-    attachment = Attachment.pending_direct_uploads.where(id: attachment_id).first
-    local_file = attachment && attachment.file.local_file
+    attachment = Attachment.pending_direct_uploads.find_by(id: attachment_id)
+    local_file = attachment&.file.local_file
```

Comment:
An attachment is guaranteed to have a file. But if the attachment is `nil` the expression `attachment&.file` will be nil and `attachment&.file.local_file` will throw a `NoMethodError: undefined method `local_file' for nil:NilClass`.

So the `attachment &&` there was in purpose. The alternative is to write `attachment&.file&.local_file`.
Which I find less clear in this case, though.

Reply:
I changed it back.